src/modules/asab-config/MicroservicesContainer.js

```diff
@@ -31,9 +34,12 @@ export default (props) => {
 
         const ASABConfigAPI = props.app.axiosCreate('asab_config');
 
-        ASABConfigAPI.get('/microservices').
-            then(res => parseDataIntoArr(res.data)).
-            then(lst => setList(lst));
+        ASABConfigAPI.get('/microservices')
+            .then(res => setList(parseDataIntoArr(res.data)))
```

Comment:
@titkov49 Here it might be also good to catch the response from the config service. If there is some error message returned from the service on `200`. You want to check that with Mithun.

Reply:
I contacted Mithun and he'll add some messages for this in next MR. For now I added handling cases of status code 200 and svc == null.